datalake/src/main/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreService.java

```diff
@@ -530,7 +530,8 @@ public boolean shouldSdxBackupBePerformed(SdxCluster cluster, boolean entitlemen
             DetailedEnvironmentResponse environmentResponse = environmentClientService.getByName(cluster.getEnvName());
             CloudPlatform cloudPlatform = CloudPlatform.valueOf(environmentResponse.getCloudPlatform());
             if (CloudPlatform.MOCK.equalsIgnoreCase(cloudPlatform.name())) {
-                return true;
+                // TODO: This is temporary as DR is blocked for Mock on Thunderhead.
+                return false;
             }
```

Comment:
is there a jira for TODO?

Reply:
also I do not get this change completely. in the title it is said that you are enabling datalakedr for integration tests and e2e tests, though you are specifying here that in case of mock (which is the platform for integrations tests) backup should not be performed

Reply:
Yes well the exact goal of this was to enable backups for the non-mock e2e tests using cloud providers such as AWS or Azure. 

However I have learned some new information which may allow me to remove this change here and add some other changes which gets the mock test working as well.